docs/guides/deployment/kubernetes.md

```diff
@@ -132,18 +134,83 @@ If you already have a registry, skip ahead to the next section.
 <!-- === "Azure"
     TODO -->
 
-## Create a work pool
+## Create a Kubernetes work pool
+
+[Work pools](/concepts/work-pools/) allow you to manage deployment infrastructure.
+We'll configure the default values for our Kubernetes base job template.
+Note that these values can be overridden by individual deployments.
+
+Let's switch to the Prefect Cloud UI, where we'll create a new Kubernetes work pool (alternatively, you could use the Prefect CLI to create a work pool).
+
+1. Click on the **Work Pools** tab on the left sidebar
+1. Click the **+** button at the top of the page
+1. Select **Kubernetes** as the work pool type
+1. Click **Next** to configure the work pool settings
+
+Let's look at a few popular configuration options.
+
+**Environment Variables**
+
+Add environment variables to set when starting a flow run.
+You can specify Python packages to install at runtime with `{"EXTRA_PIP_PACKAGES":"my_package"}`. For example `{"EXTRA_PIP_PACKAGES":"pandas==1.2.3"}` will install pandas version 1.2.3.
+Alternatively, you can specify package installation in a custom Dockerfile, which can allow you to take advantage of image caching.
+As we'll see below, Prefect can help us create a Dockerfile with our flow code and the packages specified in a `requirements.txt` file baked in.
+
+**Namespace**
+
+Set the Kubernetes namespace to create jobs within, such as `prefect`. By default, set to **default**.
+
+**Image**
+
+Specify the Docker container image for created jobs. If not set, the latest Prefect 2 image will be used.
+
+**Image Pull Policy**
+
+Select from the dropdown options to specify when to pull the image.
```

Comment:
```suggestion
Select from the dropdown options to specify when to pull the image.
When using the `IfNotPresent` policy, make sure to use unique image tags, as otherwise old images could get cached on your nodes.
```